handout/components/creating_components.md

```diff
@@ -21,6 +23,6 @@ export class Hello {
 }
 ```
 
-To use this component we simply add `<hello></hello>` to our HTML, and Angular will insert an instance of the `Hello` view between those tags.
+We need to import the **Component** decarator from '@angular/core' before we can make use of it. To use this component we simply add `<hello></hello>` to the HTML file or another template, and Angular will insert an instance of the `Hello` view between those tags.
```

Comment:
Decorator, not "decarator". "Component" and "@angular/core" should be in backticks